crates/torii/indexer/src/processors/metadata_update.rs

```diff
@@ -34,6 +34,15 @@ where
         true
     }
 
+    fn task_priority(&self) -> usize {
+        1
+    }
+
+    fn task_identifier(&self, _event: &Event) -> u64 {
+        // TODO. for now metadata updates are not parallelized
+        0
```

Comment:
Let's use a constant for the default `TaskId` where everything will be pooled if not computed.